superset/models.py

```diff
@@ -1410,7 +1421,8 @@ def visit_column(element, compiler, **kw):
 
         qry = qry.limit(row_limit)
 
-        if is_timeseries and timeseries_limit and groupby:
+        # Some SQL dialects use timeslot grouping inline with other dimensions
+        if is_timeseries and timeseries_limit and groupby and not time_groupby_inline:
```

Comment:
I guess this is because of this comment:

```
            # some sql dialects require for order by expressions
            # to also be in the select clause -- others, e.g. vertica,
            # require a unique inner alias
```

Otherwise I don't know why "some backends" need a subquery, the only thing I did is to remove the need for subquery for CH backend with a class variable.